webextension/scripts/bulk-save.js

```diff
@@ -70,11 +70,14 @@ function importAllBookmarks() {
     chrome.bookmarks.getTree((nodeTree) => {
       nodeTree.forEach((node) => {
         processTreeNode(node)
+        //Check and response for no bookmark in the browser
+        if (Object.keys(bulkSaveObj).length === 0) {
+          alert("Sorry No Bookmarks Found. Please ensure that you have bookmarks");
```

Comment:
Please run eslint before a commit to be consistent with the code styles. (e.g you have an extra `;` here)

Reply:
Sure will do that, and Thank you for guiding me